modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/GeoIpTaskState.java

```diff
@@ -160,12 +158,18 @@ public static Metadata fromXContent(XContentParser parser) {
         private final int firstChunk;
         private final int lastChunk;
         private final String md5;
+        private final boolean tar;
```

Comment:
Are all database files now provided as tarball? If so then we don't need this `tar` field?

Reply:
Right now they provide both versions, `mmdb.gz` and `.tgz`. I can filter and process only `tgz` files, then I can remove this field. We could then remove handling of simple `.gz` altogether.

Reply:
That sounds good to me. This avoids having multiple code paths that essentially have the same purpose.

Reply:
Done